src/components/Input/index.tsx

```diff
@@ -59,6 +60,9 @@ export const Input = React.forwardRef<HTMLInputElement, InputCustomProps>(
       if (handleChange) {
         handleChange(event, valueWithoutSeparator)
       }
+      if (witUpperCase) {
+        handleChange?.(event, valueWithoutSeparator.toUpperCase())
```

Comment:
is it ok that handleChange will be called twise?

Reply:
@LilitArz jan  this is Valodya :) m. resolved